src/screens/Profile/Profile.js

```diff
@@ -565,10 +633,11 @@ const mapStateToProps = ({
 
 const mapDispatchToProps = (dispatch: Function) => ({
   saveBaseFiatCurrency: (currency) => dispatch(saveBaseFiatCurrencyAction(currency)),
-  repairStorage: () => dispatch(repairStorageAction()),
```

Comment:
Same as above

Reply:
nice 👍 catch!, my editor picked it as unused, sad that flow didn't pick this up
- [ ] revoken repairStorage

Reply:
It's because you left it in Props definition, so Flow thinks it's still here. But yeah, at least linter should warn about that